### PR TITLE
Added in prod file for ICP

### PR DESCRIPTION
**Pull request by yogesh-hullatti**
https://tools.hmcts.net/jira/browse/EM-3020

### CODE REVIEW

File: k8s/prod/cluster-00/em/em-icp.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: em-icp
+    version: 0.1.10
+  values:
+    nodejs:
+      replicas: 1
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/em/icp:prod-ae3d2e63
+      ingressHost: em-icp-prod.service.core-compute-prod.internal
```
Comment (timja) [inline k8s/prod/cluster-00/em/em-icp.yaml:24]: This should be in your chart

Comment (timja) [inline k8s/prod/cluster-00/em/em-icp.yaml:24]: This should be in your chart

File: k8s/prod/cluster-00/em/em-icp.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      environment:
+        VAR_T3: trigger4
```
Comment (timja) [inline k8s/prod/cluster-00/em/em-icp.yaml:26]: This shouldn’t be needed

Comment (yogesh-hullatti) [reply k8s/prod/cluster-00/em/em-icp.yaml:26]: Thank you @timja . I will need to implement this across all our API's. Will implement this review comment as part of separate ticket.

**Pull request merged**